parser/Builder.cc

```diff
@@ -1155,13 +1165,14 @@ class Builder::Impl {
             } else {
                 loc = tokLoc(not_).join(receiver->loc);
             }
-            return make_unique<Send>(loc, transformCondition(std::move(receiver)), core::Names::bang(),
+            return make_unique<Send>(loc, transformCondition(std::move(receiver)), core::Names::bang(), tokLoc(not_),
                                      sorbet::parser::NodeVec());
         }
 
         ENFORCE(begin != nullptr && end != nullptr);
         auto body = make_unique<Begin>(tokLoc(begin).join(tokLoc(end)), sorbet::parser::NodeVec());
-        return make_unique<Send>(tokLoc(not_).join(body->loc), std::move(body), core::Names::bang(),
+        // TODO(jez) seems wrong
+        return make_unique<Send>(tokLoc(not_).join(body->loc), std::move(body), core::Names::bang(), tokLoc(not_),
                                  sorbet::parser::NodeVec());
```

Comment:
Yeah, but I don't think it ever makes sense to do the things that I would have otherwise special cased. Like if we set this, you still can't use the `methodLoc` to rename it to another method, nor show completion results, etc. I'd rather have it be zero width if you can't use it meaningfully.

Reply:
> I'd rather have it be zero width if you can't use it meaningfully.

Got it, that makes sense!